src/ja/common/message/worker_commands/set_filters.py

```diff
@@ -0,0 +1,31 @@
+"""
+This command will change the filters on the docker event loop
+"""
+from typing import Dict
+
+from ja.common.message.base import Serializable
+from ja.common.message.worker import WorkerCommand, WorkerResponse
+from ja.worker.worker_client import Worker
+
+
+class ChangeFiltersCommand(WorkerCommand):
```

Comment:
What is the purpose of this class?

Reply:
We should be able to change the [filters](https://docs.docker.com/engine/reference/commandline/events/), from the server. On what occasions the worker is going to notify the server.

Reply:
We will probably stick with a fix ones, but nevertheless I decided there could be the possibility of changing them.